### ISSUE TITLE
Stucks on generating iOS sources - MAE - v4.12.22

### ISSUE THREAD

**Issue by Pogg** [Xtraball/SiberianCMS#514]
Hi everyone,
actually I have the following problem: 
when I try to generate the iOS source (no ads) the generation process stucks and after 60 mins I get the following message in the Messages section of the Backoffice.
![image](https://user-images.githubusercontent.com/22890106/34578334-d908b9c8-f184-11e7-8336-57c0d4803cd3.png)
If I remove the blocks (as suggested by the message) and try to generate the sources again i come to the same situation. How can I solve this?
I'm using MAE - v4.12.22
Thanks for any help.

**Comment by Pogg** [comment]
**EDIT**: I found the sources in the `/var/tmp/applications/ionic/ios/<app_name>` directory on the server, manually downloaded them and used them to publish the app on App Store. All seemed to work in the publishing process, but when i loaded the build into iTunesConnect it was clear that the app's icons wheren't updated, in fact they are all 'ionic' logo, as I verified in the AppIcon.appiconset folder of the sources. Does it suggest any clue about what is failing in the ios source building process? Tnx
